FinsembleButton/FinsembleButton.jsx

```diff
@@ -123,7 +123,6 @@ export default class Button extends React.Component {
 		//If the click action has been invalidated (because the user clicked the menu Launcher while the menu was open), we allow subsequent clicks to open the menu.
 		if (!this.openMenuOnClick) {
 			this.openMenuOnClick = true;
-			return;
 		}
```

Comment:
It is used a lot https://github.com/ChartIQ/finsemble-react-controls/blob/3cc5a400e47ee749ad9613e51491b5b89e71007b/FinsembleButton/FinsembleButton.jsx#L145

And to be honest I wanted to fix the bug and didn't want to go beyond that. We could just go ahead and remove all reference to it which will generate a bigger diff.

Reply:
I am only looking at FinsembleButton.jsx.  Maybe it is used elsewhere?

But in FinsembleButton.jsx, its value is not read anywhere but here -- and removing the `return` statement makes it not have any effect besides just setting its own value.

Reply:
I spoke with Brad about this (since he wrote the code), and he confirmed that removing the `return` is a bad idea.  Instead, he suggested that we should set `openMenuOnClick` to true on a blurred event as well.  I will send you more details in a DM with videos of the desired behavior